lib/rules/declaration-property-value-no-unknown/__tests__/index.mjs

```diff
@@ -150,6 +150,12 @@ testRule({
 				}
 			`,
 		},
+		{
+			code: stripIndent`a {
+				top: /* stylelint-disable-next-line declaration-property-value-no-unknown */
+					red;
+			}`,
```

Comment:
A bit of extra test coverage in light of https://github.com/stylelint/stylelint/issues/7904